base_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super
```

Comment:
- [ ] There was an error running the tests as a result of this `super`. There is a difference between `super` and `super()`. You can refer to this [Stackoverflow answer](https://stackoverflow.com/a/31816281) for a more detailed explanation. Kindly remove this `super` or change it to `super()` to solve this error.

  ![rizo-1](https://user-images.githubusercontent.com/36542986/164205384-f159dfa7-d79e-4a9c-bdc9-fcbd81cf6720.jpg)